spec/unit/changed_rpm_files_helper_spec.rb

```diff
@@ -151,3 +151,12 @@
     )
   end
 end
+
+describe "#get_link_target" do
+  it "returns the link target" do
+    system = "double"
+    expect(system).to receive(:run_command).and_return("/etc/foo\n")
```

Comment:
Wouldn't it make sense to check that run_command is called with "readlink" as argument?